tensorboard/components/tf_backend/requestManager.ts

```diff
@@ -221,18 +230,44 @@ export class RequestManager {
       }
     });
   }
+}
+  
+function buildXMLHttpRequest(methodType: HttpMethodType, url: string,
+                             withCredentials? : boolean,
+                             contentType? : string): XMLHttpRequest {
+  const req = new XMLHttpRequest();
+  req.open(methodType, url);
+  if (withCredentials) {
+    req.withCredentials = withCredentials;
+  }
+  if (contentType) {
+    req.setRequestHeader('Content-Type', contentType);
+  }
+  return req;
+}
 
-  private formDataFromDictionary(postData: {[key: string]: string}) {
-    const formData = new FormData();
-    for (let postKey in postData) {
-      if (postKey) {
-        // The linter requires 'for in' loops to be filtered by an if
-        // condition.
-        formData.append(postKey, postData[postKey]);
-      }
+export function requestOptionsFromPostData(postData?: {[key: string]: string}):
+    RequestOptions {
+  const result = new RequestOptions();
+  if (!postData) {
+    result.methodType = HttpMethodType.GET;
+    return result;
+  }
+  result.methodType = HttpMethodType.POST;
+  result.body = formDataFromDictionary(postData);
```

Comment:
Can we set the contentType explictly to multipart form data?

Reply:
I think this done automatically (from my experiments), and I'd rather not change the behavior from what it used to be.